### PR TITLE
fix: tx list

### PR DESCRIPTION
**Pull request by nitinmittal23**

### CODE REVIEW

File: packages/example/src/components/ui/molecule/TransactionsList/TransactionListItem/TransactionListItem.view.tsx [hunk 1]
```diff
@@ -13,71 +13,45 @@ import {
...
+import { getIcon, getTxnToFromLabel, getTxnValues } from './types';
 
 interface Props {
   transaction: Transaction;
 }
 
 export const TransactionListItemView = ({ transaction }: Props) => {
   const wallet = useAppSelector((state) => state.wallet);
-  const [currencySymbol, setCurrencySymbol] = useState('N/A');
+  const [currencySymbol, setCurrencySymbol] = useState('AVL');
```
Comment (vibhurajeev) [inline packages/example/src/components/ui/molecule/TransactionsList/TransactionListItem/TransactionListItem.view.tsx]: Need to change this to AVAIL

File: packages/example/src/components/ui/molecule/TransactionsList/TransactionListItem/TransactionListItem.stories.tsx [hunk 2]
```diff
@@ -7,22 +7,7 @@ export default {
...
-  contractCallData: [
-    '0x6b686ebe2cbd70b37b54df1b9889cc3095b55f386110843912efcaed416ff3f',
-    '0x0de0b6b3a7640000'
-  ],
-  timestamp: 1655705597,
-  status: 'Accepted on L1',
-  eventIds: ['245417_20_0'],
-  failureReason: ''
-};
+const transaction: Transaction = {} as Transaction;
```
Comment (vibhurajeev) [inline packages/example/src/components/ui/molecule/TransactionsList/TransactionListItem/TransactionListItem.stories.tsx:10]: Why is this empty object and assertion okay here?

File: packages/example/src/utils/utils.ts [hunk 3]
```diff
@@ -44,6 +44,7 @@ export const getHumanReadableAmount = (asset: Erc20TokenBalance) => {
   //   ? assetAmount
   //   : ethers.utils.formatUnits(asset.amount, asset.decimals);
   if (asset.amount > 0) {
+    console.log(asset.amount);
```
Comment (vibhurajeev) [inline packages/example/src/utils/utils.ts]:
```suggestion
```

File: packages/example/src/utils/utils.ts [hunk 4]
```diff
@@ -56,6 +57,7 @@ export const getHumanReadableAmount = (asset: Erc20TokenBalance) => {
     if (firstNonZeroIndex === -1) {
       return integerPart;
     }
+    console.log('aaaaaaaa');
```
Comment (vibhurajeev) [inline packages/example/src/utils/utils.ts]:
```suggestion
```

File: packages/example/src/components/ui/molecule/TransactionsList/TransactionListItem/TransactionListItem.stories.tsx (hunk 2 above)
Comment (nitinmittal23) [reply packages/example/src/components/ui/molecule/TransactionsList/TransactionListItem/TransactionListItem.stories.tsx:10]: these are stories. you can ignore. consider it like tests

**Pull request merged**